hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmSnapshotManager.java

```diff
@@ -628,7 +628,7 @@ public ReferenceCounted<IOmMetadataReader> getActiveFsMetadataOrSnapshot(
       // Updating the volumeName & bucketName in case the bucket is a linked bucket. We need to do this before a
       // permission check, since linked bucket permissions and source bucket permissions could be different.
       ResolvedBucket resolvedBucket = ozoneManager.resolveBucketLink(Pair.of(volumeName,
-          bucketName), false);
+          bucketName), false, false);
```

Comment:
Thanks @swamirishi for the patch.

As long as we make sure `getActiveFsMetadataOrSnapshot()` can ONLY be called by a privileged user that has admin permission, I am good with this. According to your updated PR description that seems to be the case here.

Because otherwise it becomes a potential security hole (info leak).

Reply:
Please update the above comment since we are skipping ACL check now.